### ISSUE TITLE
[Suggestion] Migrations DDL

### ISSUE THREAD

**Issue by anothergituser** [laravel/framework#21085]
- Laravel Version: 5.4
- PHP Version: 7.1
- Database Driver & Version: 10.1.21-MariaDB

Like the title says it, it's a feature request

When running make:migration i would like to have 2 more files generated, with the same name as the migration file and the suffix _up_ddl and _down_ddl for the up and down functions of the migration. Initially they would be empy just like the migration file but upon running ./artisan migrate they would be populated with the DDL generated by the --pretend option. Maybe they could live in the database/migrations/ddl folder or in the same database/migrations folder next to their migration file.

I propose this because normally i don't run artisan migrate or artisan migrate:rollback in production and i always need to keep the ddl for each migration (up and down) when i need to update the production db or rollback changes in production and this would automate the process.

Maybe it could be an option for make:migration like --ddl and it would do the above.

Thanks.

**Comment by Dylan-DPC** [comment]
I don't think this is something that everyone needs. If you find a frequent need for it, make a package for it. Also this repo is for bug tracking. If you want to make a proposal you can move to laravel/internals repo

**Comment by themsaid** [comment]
For suggestions and feature requests please use the https://github.com/laravel/internals repo.
